### PR TITLE
address copy button added

### PR DESCRIPTION
**Pull request by ijlal321**
### **Feature Added: Copy Address Button in Accounts Dropdown**

**Reason:**  
Previously, to copy an address, users needed to select the address, copy it, and then reselect the previous address. The goal of this feature is to streamline the process by providing a button that allows users to directly copy the address, eliminating the extra steps.

**Implementation Details:**  
Originally this dropdown uses a html native select (which does not support any buttons inside option ). This is now replaced by react-select component. React-select allows a dropdown option to have buttons.

**Code Dependability:**  
This new select is made to look exactly as previous one. Most classes and components used are same as previous select. The copy button is imported from remix-copy. All fonts are working perfectly.

### CODE REVIEW

**Comment by Aniket-Engg** [comment]
@ijlal321 linting is failing for the PR

**Comment by ijlal321** [comment]
> @ijlal321 linting is failing for the PR

Got it, fixed it 👍

File: libs/remix-ui/run-tab/src/lib/components/account.tsx [hunk 1]
```diff
@@ -196,6 +197,77 @@ export function AccountUI(props: AccountProps) {
     )
   }
 
+  const customStyles = {
```
Comment (LianaHus) [inline libs/remix-ui/run-tab/src/lib/components/account.tsx:200]: please do not use custom styles

**Comment by LianaHus** [comment]
please check out how the custom dropdown is implemented in **Environment** dropdown and do not use custom styles.

**Comment by Aniket-Engg** [comment]
Closing because of inactivity.

**Pull request closed without merging**